src/templates/PoolV2/Hero/ShareImageModal/index.tsx

```diff
@@ -77,13 +93,15 @@ const ShareImageModal = ({
 
           const file = canvas.toDataURL('image/png')
           const id = url.split('/').pop()
+          const _id = id?.split('?')[0]
```

Comment:
Eu acho que quando a gnt trabalha com url a gnt pode usar a classe da própria `URL`, como:

```suggestion
          const id = new URL(url).pathname.split('/').at(-1)
```

La embaixo removeria o `_id` pra `id`